src/pspm_combine_markerchannels.m

```diff
@@ -8,17 +8,16 @@
 % ● Format
 %   [sts, outchannel] = pspm_combine_markerchannels(datafile, options)
 % ● Arguments
-%    datafile:          data file name(s) (char, or cell array for 
-%                       multiple files)
+%    datafile:          data file name(s): char
```

Comment:
Just to confirm my understanding. Loading multiple files is no longer supported after this update. Only one file is allowed to be imported.